includes/wikia/models/DesignSystemGlobalFooterModel.class.php

```diff
@@ -1,9 +1,9 @@
 <?php
 
-class DesignSystemGlobalFooterModel extends WikiaModel {
+abstract class DesignSystemGlobalFooterModel extends WikiaModel {
 	const DEFAULT_LANG = 'en';
 
-	private $hrefs = [
+	protected $hrefs = [
 		'default' => [
 			'fan-communities' => 'http://fandom.wikia.com/explore',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "fan-communities" 13 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)